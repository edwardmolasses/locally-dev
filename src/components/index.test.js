/* eslint-env node, jest */

import { h } from "preact";
import render from "preact-render-to-string";
import Locally from "./index";

describe("Locally Snapshot", () => {
  it("should render header with content", () => {
    // console.log(document.body);
    const tree = render(<Locally upc={"686487414242"} countryCode={"en-CA"} root={"locally-widget-root"} />);
    expect(tree).toMatchSnapshot();
  });
});
