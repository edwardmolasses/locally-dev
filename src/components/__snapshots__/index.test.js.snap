// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Locally Snapshot should render header with content 1`] = `""`;
